doc/contributor-how-to/contribute-layers.md

```diff
@@ -6,18 +6,27 @@ real ones. NOTE: When generating templates, but _not_ when fetching, you can
 use `./scripts/experimental/local_cli.sh config-template <dataset|layer>` in
 place of the `cli.sh` commands.
 
+## Add a dataset
+
 If the layer does not use an existing dataset, start with a new dataset.
 
 ```
 ./scripts/cli.sh config-template dataset > \
   qgreenland/config/datasets/new_dataset.py
 ```
 
+## Fetch the data
+
 Test the dataset by fetching the data:
 
 ```
 ./scripts/cli.sh fetch new_dataset_id
 ```
+NOTE: If your fetch command results in an error, there may be issues with 
```

Comment:
Lets turn this into a `note` admonition, e.g,

\```{note}
If your fetch command results in an error, there may be issues with
the entry of your dataset. Go back to your dataset.py file and 
make sure that all fields are filled in (abstract, title, etc.) to avoid linting errors.
\```